app/helpers.js

```diff
@@ -0,0 +1,295 @@
+import PIXI from '../bower_components/pixi.js/bin/pixi';
+
+// TODO: This file was created by bulk-decaffeinate.
+// Sanity-check the conversion and remove this comment.
+/*
+ * decaffeinate suggestions:
+ * DS101: Remove unnecessary use of Array.from
+ * DS102: Remove unnecessary code created because of implicit returns
+ * DS207: Consider shorter variations of null checks
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
+// [a,b,c,d].remove(1) => [a,c,d]
+// [a,b,c,d].remove(0,2) => [d]
+Array.prototype.remove = function(from, to) {
```

Comment:
It would be good to eliminate these `Array.prototype` extensions at some point, but since they're also a preexisting condition it shouldn't hold up the PR.